circuits/cpp/src/aztec3/circuits/kernel/private/private_kernel_circuit.cpp

```diff
@@ -27,22 +27,16 @@ using aztec3::circuits::silo_nullifier;
 
 // TODO: NEED TO RECONCILE THE `proof`'s public inputs (which are uint8's) with the
 // private_call.call_stack_item.public_inputs!
-CT::AggregationObject verify_proofs(Composer& composer,
-                                    PrivateKernelInputsInner<CT> const& private_inputs,
-                                    size_t const& num_private_call_public_inputs,
-                                    size_t const& num_private_kernel_public_inputs)
+CT::AggregationObject verify_proofs(Builder& builder, PrivateKernelInputsInner<CT> const& private_inputs)
 {
     // compute P0, P1 for private function proof
-    CT::AggregationObject aggregation_object = Aggregator::aggregate(
-        &composer, private_inputs.private_call.vk, private_inputs.private_call.proof, num_private_call_public_inputs);
+    CT::AggregationObject aggregation_object =
```

Comment:
This is nice, `aggregate` shouldn't have to know what the number of public inputs of the private call is.